src/onboarding/Onboarding.js

```diff
@@ -2,6 +2,8 @@ import React from 'react'
 import styled from 'styled-components'
 import { Motion, spring } from 'react-motion'
 import { spring as springConf } from '@aragon/ui'
+import { isNameUsed } from '@aragon/wrapper'
```

Comment:
I think we've tried to scope everything aragon.js related to `aragonjs-wrapper.js`, so it'd be nice just to re-export this from there.